components/organisms/ReportAProblem.js

```diff
@@ -268,48 +83,220 @@ export function ReportAProblem(props) {
             </li>
             <li className="text-xs sm:text-sm font-body my-4 leading-tight sm:leading-6">
               <b>
-                {t("reportAProblemNoPersonalInfo", { lng: props.language })}
-              </b>
-              ,&nbsp;
-              {t("reportAProblemNoPersonalInfoDetails", {
-                lng: props.language,
-              })}
-            </li>
-            <li className="text-xs sm:text-sm font-body my-4 leading-tight sm:leading-6">
-              <b>{t("reportAProblemMoreInfo", { lng: props.language })}</b>
-              ,&nbsp;
-              {t("reportAProblemMoreInfoDetails", { lng: props.language })}
-              &nbsp;
-              <a
-                className="underline text-xs sm:text-sm font-body hover:text-canada-footer-hover-font-blue text-canada-footer-font"
-                href={t("reportAProblemMoreInfoLink", {
+                {t("reportAProblemFeedbackConfidential", {
                   lng: props.language,
                 })}
-              >
-                {t("reportAProblemMoreInfoLinkText", { lng: props.language })}
-              </a>
+              </b>
             </li>
           </ul>
-
-          <a
-            className="underline text-xs sm:text-sm font-body hover:text-canada-footer-hover-font-blue text-canada-footer-font my-4 underline"
-            href={t("reportAProblemPrivacyStatementLink", {
-              lng: props.language,
-            })}
-          >
-            {t("reportAProblemPrivacyStatement", { lng: props.language })}
-          </a>
-          <ActionButton
-            id="submit"
-            className="rounded block mt-4"
-            type="submit"
-            dataCy="report-a-problem-submit"
-            dataTestId="report-a-problem-submit"
-            analyticsTracking
+          <div className="mb-6">
+            <a
+              className="underline text-xs sm:text-sm font-body hover:text-canada-footer-hover-font-blue text-canada-footer-font"
+              href={t("reportAProblemPrivacyStatementLink", {
+                lng: props.language,
+              })}
```

Comment:
We have our own privacy statement page, should this link get changed to that instead on canada.ca's privacy statement?

Reply:
> We have our own privacy statement page, should this link get changed to that instead on canada.ca's privacy statement?

That's a good question, let me confirm with the design team.

Reply:
The privacy link is now using the one on Alpha site